src/app/AuthProvider.tsx

```diff
@@ -53,8 +56,8 @@ export const AuthProvider = ({children}: { children: React.ReactNode }) => {
       window.location.assign(`${keycloakConfig.url}/realms/${keycloakConfig.realm}/protocol/openid-connect/auth` +
           `?client_id=${keycloakConfig.clientId}&redirect_uri=${currentUrl}&response_type=code&scope=openid`);
     }
-
-  }, [handleNotAuthenticated, router, user]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Skal bare kjøre på oppstart? Kanskje se på andre lifecycle events. Kan vi bruke `componentDidMount`?

Reply:
Lifecycle-metoder kan ikke lengre brukes i funksjonskomponenter, så denne måten med tom dependency array tilsvarer `componentDidMount`. Alternativ for å kjøre kun en gang er å flytte metoden ut av komponenten, men da sliter man med å sette state.